iroh-gateway/src/templates.rs

```diff
@@ -1,19 +1,28 @@
-pub const DIR_LIST: &str = "
-<div>
-    <ol>
-        {{#each links}}
-        <li>
-            <a href=\"{{this.path}}\">{{this.name}}</a>
-        </li>
-        {{/each}}
-    </ol>
-</div>";
+use lazy_static::lazy_static;
+use std::{collections::HashSet, ffi::OsStr, path::Path};
 
-pub const NOT_FOUND: &str = "
-<div>
-    <h1>404 Not Found</h1>
-    <p>
-        The requested resource was not found.
-    </p>
-</div>
-";
+pub const DIR_LIST_TEMPLATE: &str = include_str!("../assets/dir_list.html");
+pub const NOT_FOUND_TEMPLATE: &str = include_str!("../assets/404.html");
+pub const STYLESHEET: &str = include_str!("../assets/style.css");
+pub const ICONS_STYLESHEET: &str = include_str!("../assets/icons.css");
+
+lazy_static! {
+    static ref KNOWN_ICONS: HashSet<&'static str> = HashSet::from([
```

Comment:
there is a nicer version that avoids lazy_static: https://docs.rs/phf for this use case (also faaster ;))